lib/live-stream.js

```diff
@@ -53,6 +55,11 @@ function streamData (args, archives, cb) {
               cb()
             })
           })
+          dat.hyperStats = Stats({
+            archive: dat.archive,
+            db: sub(args.db, `${encoding.encode(dat.key)}-stats`)
+          })
+          dat.hyperStats.on('update', () => cb())
```

Comment:
Perhaps out of scope for this patch, but this pattern feels like a smell - now that we've got dat-js in here we should think of a way to actually move data around (: